lib/phonemiser.py

```diff
@@ -1,7 +1,9 @@
-# Functions for doing grapheme to phoneme for supported languages
-# The default functions first do LRLM segmentation of the orthography
-# and then lookup the segments in a lookup table. Any symbols which
-# do not appear are discarded.
+"""Functions for doing grapheme to phoneme for supported languages The default.
+
+functions first do LRLM segmentation of the orthography and then lookup
```

Comment:
You should include the long form of this acronym here as it's likely the first place the reader will see it in this documentation.

Reply:
done.